File: JS/02.Basics/01.3-FirstStepsinCodingExercise/07.FoodDelivery.js
function calculateOrderPrice(chickenMenus, fishMenus, vegetarianMenus) {
    const chickenMenuPrice = 10.35;
    const fishMenuPrice = 12.40;
    const vegetarianMenuPrice = 8.15;

    const chickenMenusTotal = chickenMenus * chickenMenuPrice;
    const fishMenusTotal = fishMenus * fishMenuPrice;
    const vegetarianMenusTotal = vegetarianMenus * vegetarianMenuPrice;

    const totalMenuPrice = chickenMenusTotal + fishMenusTotal + vegetarianMenusTotal;

    const dessertPrice = 0.2 * totalMenuPrice;
    const deliveryCost = 2.50;

    const totalOrderPrice = totalMenuPrice + dessertPrice + deliveryCost;

    const formattedTotalOrderPrice = totalOrderPrice.toFixed(2);

    console.log(formattedTotalOrderPrice);
}

calculateOrderPrice(2, 4, 3);
calculateOrderPrice(9, 2, 6);

File: C++/01-Basics/01.3-FirstStepsinCodingExercise/07.FoodDelivery/07.FoodDelivery.cpp
#include <iostream>
#include <iomanip>

using namespace std;

int main() {
    int chickenMenus, fishMenus, vegetarianMenus;

    cin >> chickenMenus >> fishMenus >> vegetarianMenus;

    double chickenMenuPrice = chickenMenus * 10.35;
    double fishMenuPrice = fishMenus * 12.40;
    double vegetarianMenuPrice = vegetarianMenus * 8.15;

    double totalMenuPrice = chickenMenuPrice + fishMenuPrice + vegetarianMenuPrice;
    double dessertPrice = 0.20 * totalMenuPrice;
    double deliveryPrice = 2.50;

    double totalPrice = totalMenuPrice + dessertPrice + deliveryPrice;

    cout << totalPrice << endl;

    return 0;
}